sound/soc/intel/boards/sdw_rt711_rt1308_rt715.c

```diff
@@ -80,8 +81,8 @@ struct codec_info {
 		     bool playback);
 };
 
-#if IS_ENABLED(CONFIG_SND_SOC_SOF_HDA_COMMON_HDMI_CODEC)
-static struct snd_soc_jack hdmi[3];
+#if IS_ENABLED(CONFIG_SND_HDA_CODEC_HDMI)
```

Comment:
This is conflict with https://github.com/thesofproject/linux/pull/1794. And we already [depends on SND_HDA_CODEC_HDMI](https://github.com/thesofproject/linux/blob/topic/sof-dev/sound/soc/intel/boards/Kconfig#L575) on Kconfig.

Reply:
@bardliao Ack, let's go with the approach with #1794 .
@RanderWang Once 1794 is merged, I think this patch is still needed. You could make similar change as I did in #1777 and make HDMI optional at runtime.